nni/compression/pytorch/speedup/v2/utils.py

```diff
@@ -79,3 +80,15 @@ def tree_map_zip(fn: Any, *pytrees):
             spec_list.append(spec)
         assert all(len(args) == len(flat_args_list[0]) for args in flat_args_list), 'Inconsistent tree nodes length.'
         return tree_unflatten([fn(*args) for args in zip(*flat_args_list)], spec_list[0])
+
+
+def poss_deepcopy(o, logger: logging.Logger = None) -> Any:
```

Comment:
is this for `next()`?

Reply:
yes, it can fix `next` issue, and also for some other none-tensor inplace-change issue

Reply:
nice! but I guess poss_copy might be an unclear function name?